services/src/main/java/org/jboss/windup/web/services/rest/RegisteredApplicationEndpointImpl.java

```diff
@@ -20,9 +20,12 @@
     private RegisteredApplicationService registeredApplicationService;
 
     @Override
-    public Collection<RegisteredApplication> getAllApplications()
+    public Collection<RegisteredApplication> getAllApplications(Long projectId)
     {
-        return this.registeredApplicationService.getAllApplications();
+        if (projectId == null)
```

Comment:
To clarify, I am for such path, perhaps shorter one, like, `.../id-{id}`. It's only it was already this way. Maybe we should standardize the way we do the endpoint URLs.